DEPLOY.md

```diff
@@ -0,0 +1,44 @@
+# Deploying Atlas
+
+This is a guide to getting the Atlas smart contracts deployed on an EVM chain.
+
+## Atlas Deployment Process
+
+1. Check that the contracts compile and all tests pass.
+
+```bash
+forge test
+```
+
+2. Set up your `.env` file with the variables relevant to the chain to which you are trying to deploy. Note: the `GOV_PRIVATE_KEY` is the private key to the public address from which the contracts will be deployed. You must also set the `DEPLOY_TO` variable to a valid chain option: `SEPOLIA`, `MAINNET`, `AMOY`, `POLYGON`, or `LOCAL`. In this example we would be deploying to Polygon mainnet.
```

Comment:
Consider linking to Wallet documentation Davide has created here.

Reply:
Added here https://github.com/FastLane-Labs/atlas/pull/378/commits/207442612fb0058d94060d74e5bbeac191f37915